### PR TITLE
Remove `.vscode`

### LINKED ISSUE #1328: Remove `.vscode/settings.json`...
...and maybe even `launch.json`, giving devs the ability to control their workspace. This issue is similar to https://github.com/PolicyEngine/policyengine-app/issues/1118.

We could add `.vscode` to `.gitignore` or rename the files to something like `settings_template.json`.

### PR DESCRIPTION
**Pull request by abhcs**
Fix #1328 by 
- removing `.vscode`, and
- adding `.vscode` to `.gitignore`.

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- bump: patch
+  changes:
+    removed:
+    - .vscode folder
```
Comment (MaxGhenis) [inline changelog_entry.yaml:4]: and below, and newline at the bottom may fix the CI
```suggestion
      - .vscode folder
```

**Comment by anth-volk** [comment]
I believe this may be failing for the same reason as #1380 does. @abhcs, would you be willing to open this PR again off of a clone branch instead of a fork?

In the future, we'll need to make clear that this repo should be cloned.

**Comment by abhcs** [comment]
I tried pushing a clone branch but got the following error:

```
ERROR: Permission to PolicyEngine/policyengine-api.git denied to abhcs.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by MaxGhenis** [comment]
Just invited you for write access, could you try again?

**Comment by abhcs** [comment]
Closing in favor of #1468.

**Pull request closed without merging**